src/compiler/commandLineParser.ts

```diff
@@ -481,6 +481,12 @@ namespace ts {
                 name: "plugin",
                 type: "object"
             }
+        },
+        {
+            name: "checkJsFiles",

```

Comment:
Since we have `allowJs`, shouldn't this be `checkJs`? Having 'Files' on just one of them seems inconsistent.